packages/transformers/js/src/JSTransformer.js

```diff
@@ -472,22 +483,57 @@ export default (new Transformer({
         }
       } else if (dep.kind === 'URL') {
         asset.addURLDependency(dep.specifier, {
+          bundleBehavior: 'isolated',
           loc: convertLoc(dep.loc),
         });
       } else if (dep.kind === 'File') {
         asset.invalidateOnFileChange(dep.specifier);
       } else {
-        if (dep.kind === 'DynamicImport' && isURL(dep.specifier)) {
```

Comment:
This was broken already because we already replaced the `import()` with a `require()` in rust. The isUrl check was interfering with usage of pipelines in dynamic import.